inst/emails/contact_initial_ans

```diff
@@ -2,15 +2,19 @@ Dear %s,
 
-At this time, please note that NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are often required to be submitted on that same timeframe (although the timing and submission requirements for data varies between programs). More detailed information about what will be expected can be found at https://arcticdata.io/submit/#who-must-submit.
+NSF requires all metadata created during your project be submitted to a national data center or another long-lived, publicly accessible archive within two years of collection or before the end of your award, whichever comes first. Additionally, data and derived data products are often required to be submitted on that same timeframe. More detailed information about what will be expected can be found at: https://arcticdata.io/submit/#who-must-submit.
 
-With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Typically, NSF researchers already do a great job of collecting data and metadata, but you will want to ensure your data and metadata are collected in a manner that can be easily accessed and understood by other researchers after it is published. We recommend that you visit our data submission guidelines page (https://arcticdata.io/submit/) to get a better understanding of what is expected for submissions to the Arctic Data Center.
+With these requirements in mind, we strongly recommend that you plan to keep detailed and well-documented data and metadata records throughout your project. Some guidelines are provided here: https://arcticdata.io/submit/#organizing-your-data. If you are interested, we also have annual training opportunities posted here: https://arcticdata.io/training/.
 
-If you have any questions about the Arctic Data Center or NSF requirements, please reply to this email and we will respond as soon as possible. We will also contact you again near the end of your award with reminders about the data submission requirements.
+If your research will involve large data submissions (a total size of 500MB or more), please get in touch with us so we can create a plan together for those datasets.
+
+We also wanted to highlight the data portals feature at the Arctic Data Center. If your project involves multiple related datasets, such as those that are part of a larger research project or organization, a data portal can allow datasets to be discovered together. More information on how to create a data portal and examples can be found here: https://arcticdata.io/data-portals/.
```

Comment:
This is good, but we should add more from our promotional language describing the data portals feature, and give a couple of examples (e.g., a portal for a single grant, a portal for multiple groups collaborating across time on multiple grants, and thematic portals on specific topics (e.g., Salmon) or locations (e.g., Summit Station). Maybe borrow some language from the data-portals page.

Reply:
there were more examples but I opted for less text because the emails were approaching 2 pages and I was worried about the recipient tuning out most of it if we came with a wall of text. I'll play around with the text to see if I can get a balance of both